src/components/audio-controls.tsx

```
@@ -64,25 +64,27 @@ export function AudioControls() {
-                d="M15.536 8.464a5 5 0 010 7.072m2.828-9.9a9 9 0 010 12.728M5.586 15.414a2 2 0 001.414.586h5.5a2 2 0 001.414-.586l4-4a2 2 0 000-2.828l-4-4a2 2 0 00-2.828 0l-4 4a2 2 0 00-.586 1.414v5.5z"
-              />
-            </svg>
-          </div>
-          <div className="flex-1">
-            <h3 className="font-medium text-white">Audio Alerts Locked</h3>
-            <p className="text-sm text-gray-400">Click to enable alert sounds</p>
+      <div className="rounded-2xl border border-orange-500/20 bg-[#0a0a0f] p-4">
+        <div className="flex flex-col gap-4 sm:flex-row sm:items-center">
+          <div className="flex items-center gap-3">
+            <div className="text-orange-500">
+              <svg className="h-5 w-5" fill="none" viewBox="0 0 24 24" stroke="currentColor">
```

Biome (2.4.7)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)